src/reducers/books.js

```diff
@@ -0,0 +1,45 @@
+const initialBookState = {
+  books: [
+    {
+      id: Math.floor(Math.random() * 10000000),
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)